panel.py

```diff
@@ -245,6 +245,10 @@ def __init__(self, name: str, fdt: Fdt2, node: int) -> None:
 		if fdt.getprop_or_none(node, 'qcom,mdss-dsi-tx-eot-append') is None:
 			self.flags.append('MIPI_DSI_MODE_NO_EOT_PACKET')
 
+		self.cphy_mode = False
+		if fdt.getprop_or_none(node, 'qcom,panel-cphy-mode') is not None:
+			self.cphy_mode = True
```

Comment:
`self.cphy_mode = fdt.getprop_or_none(node, 'qcom,panel-cphy-mode') is not None`

Reply:
ok, I have completed it.